jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TabSection.java

```diff
@@ -0,0 +1,62 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.ButtonToggle;
+import com.epam.jdi.light.angular.elements.complex.Tabs;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+
+public class TabSection {
+
+    /* BASIC TABS */
+    @UI("#basic-tab")
+    public Tabs basicTabsSection;
```

Comment:
To my mind, 'basicTabs' rather than 'basicTabsSection' is an appropriate name here.